src/PrestaShopBundle/Controller/Admin/Order/DeliveryController.php

```diff
@@ -0,0 +1,131 @@
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+namespace PrestaShopBundle\Controller\Admin\Order;
+
+use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
+use PrestaShopBundle\Exception\FileUploadException;
+use PrestaShopBundle\Form\Admin\Order\Delivery\SlipOptionsType;
+use PrestaShopBundle\Security\Annotation\AdminSecurity;
+use PrestaShopBundle\Security\Voter\PageVoter;
+use PrestaShop\PrestaShop\Adapter\Order\Invoice;
+use PrestaShop\PrestaShop\Core\Form\FormHandler;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\Form\Form;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Admin controller for the Order Delivery
+ */
+class DeliveryController extends FrameworkBundleAdminController
+{
+    /**
+     * Main page for Delivery slips.
+     *
+     * @Template("@PrestaShop/Admin/Order/Delivery/slip.html.twig")
+     * @AdminSecurity("is_granted(['read', 'update', 'create', 'delete'], request.get('_legacy_controller')~'_')", message="Access denied.")
+     *
+     * @param Request $request
+     *
+     * @return array|RedirectResponse
+     */
+    public function slipAction(Request $request)
+    {
+        /* @var $formHandler FormHandler */
+        $formHandler = $this->get('prestashop.adapter.order.delivery.slip.options.form_handler');
+        /* @var $form Form */
+        $form = $formHandler->getForm();
+
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
+            $errors = $formHandler->save($form->getData());
+            if (empty($errors)) {
+                $this->addFlash(
+                    'success',
+                    $this->trans('Update successful', 'Admin.Notifications.Success')
+                );
+            } else {
+                $this->flashErrors($errors);
+            }
+
+            return $this->redirectToRoute('admin_order_delivery_slip');
+        }
+
+        return [
+            'optionsForm' => $form->createView(),
+            'pdfForm' => $this->get('prestashop.adapter.order.delivery.slip.pdf.form_handler')->getForm()->createView(),
+            'help_link' => $this->generateSidebarLink($request->attributes->get('_legacy_controller')),
+            'layoutTitle' => $this->trans('Delivery Slips', 'Admin.NavigationMenu'),
```

Comment:
Looks like one full stop is missing: _Admin.Navigation.Menu_

Reply:
Woot, nice catch, Thanks a lot =)